oioioi/liveranking/templates/liveranking/autoDonuts.html

```diff
@@ -69,8 +92,11 @@
 </head>
 <body>
     <div id="header"></div>
-    <div id="donut1"></div>
-    <div id="donut2"></div>
+    <div id="info">
+        <div id="text"></div>
```

Comment:
is this for the trivia? :thinking:

Reply:
Exactly, it was mentioned in the original commit "add a #text div in autoDonuts, which will be useful for displaying trivia"

Reply:
is the trivia code still in, or was it axed? I liked that feature but I think we lost the list of trivia messages :/

Reply:
Since trivia code got lost so will be this change